src/test/java/JdiHomework/main/entities/MetalsAndColorsDataset.java

```diff
@@ -0,0 +1,66 @@
+package JdiHomework.main.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+import java.util.Arrays;
+
+public class MetalsAndColorsDataset extends DataClass<MetalsAndColorsDataset> {
+    public int[] summary;
+    public int sum;
+    public String[] elements;
+    public String color;
+    public String metals;
+    public String[] vegetables;
+
+    public int[] getSummary() {
+        return summary;
+    }
+
+    public int getSum(){
+        if(this.summary != null) return this.sum = this.summary[0] + this.summary[1];
```

Comment:
I don't get it. This statement means that if (this.summary **==** null) getSum method should return null ??